src/main/java/org/cardanofoundation/cfexploreraggregator/addresstxcount/processor/AddressTxCountProcessor.java

```diff
@@ -102,9 +102,12 @@ public void handleCommitEvent(CommitEvent commitEvent) {
     }
 
     private void handleAddressAggregation(CommitEvent commitEvent, Map<String, Tuple<Long, Long>> snapshot) {
+        List<AddressTxCountEntity> allByAddressIn = addressTxCountRepository.findAllByAddressIn(snapshot.keySet().stream().toList());
```

Comment:
I'm gonna be pendantic, but:
* please rename into something more meaningful
* would passing a set work the same? if set works, we're unnecessarily creating throwaway stuff which will put pressure on GC and overall slow down sync

Reply:
Right, Set works also. Good point!